packages/host/app/components/operator-mode/code-mode.gts

```diff
@@ -395,10 +381,33 @@ export default class CodeMode extends Component<Signature> {
     return this.moduleContentsResource?.declarations || [];
   }
 
+  private get _selectedDeclaration() {
+    return this.moduleContentsResource?.declarations.find((dec) => {
+      // when refreshing module,
+      // checks localName from serialized url
+      if (
+        this.operatorModeStateService.state.codeSelection.localName ===
+        dec.localName
+      )
+        return true;
+
+      // when opening new definition,
+      // checks codeRef from serialized url
+      let codeRef = this.operatorModeStateService.state.codeSelection?.codeRef;
+      if (isCardOrFieldDeclaration(dec) && codeRef) {
+        return (
+          dec.exportedAs === codeRef.name || dec.localName === codeRef.name
+        );
+      }
+      return false;
+    });
+  }
+
```

Comment:
This function resolves what declaration is selected depending on
- if the user has been choosing the declaration
- if the user navigated from another module 

We get this info from the serialized url. This is so we we can recover selection state that is lost by a possible loader refresh

Reply:
Just real quick I see that you have `if` without `{}`. Prettier should have fixed that. Can you merge upstream to get the prettier fix and make sure you run it on all your changed files.